app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/test/chapter11/tests/EmailToShareListTest.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.android.architecture.blueprints.todoapp.test.chapter11.tests
+
+import android.support.test.rule.ActivityTestRule
+import com.example.android.architecture.blueprints.todoapp.settings.SettingsActivity
+import com.example.android.architecture.blueprints.todoapp.test.chapter11.screens.GeneralScreen
+import com.example.android.architecture.blueprints.todoapp.test.chapter11.screens.SettingsScreen
+import org.junit.*
+import org.junit.Assert.assertTrue
+
+class EmailToShareListTest {
+
+    private val settingsScreen = SettingsScreen()
+    private val generalScreen = GeneralScreen()
+
+    @get:Rule
+    var activityTestRule = ActivityTestRule(SettingsActivity::class.java)
+
+    @Test
+    fun cancelEmailToShareToDoListOption() {
+        SettingsScreen()
```

Comment:
This initialization is not used later in any way - please remove it and avoid dead code.